curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-roman-numeral-converter-project/build-a-roman-numeral-converter.md

```diff
@@ -163,6 +163,34 @@ convertBtnEl.click();
 assert.strictEqual(outputEl.innerText.trim(), 'MMMCMXCIX');
 ```
 
+When the `#number` element contains a random negative number and the `#convert-btn` element is clicked, the `#output` element should contain the text `"Please enter a number greater than or equal to 1"`.
+
+```js
+const numberInputEl = document.getElementById('number');
+const convertBtnEl = document.getElementById('convert-btn');
+const outputEl = document.getElementById('output');
+
+const randomNegativeNumber = Math.floor(Math.random() * -4000) - 2; 
+
+numberInputEl.value = randomNegativeNumber;
+convertBtnEl.click();
+assert.strictEqual(outputEl.innerText.trim().replace(/[.,?!]+$/g, '').toLowerCase(), 'please enter a number greater than or equal to 1');
+```
+
+When the `#number` element contains a number greater than 4000 and the `#convert-btn` element is clicked, the `#output` element should contain the text `"Please enter a number less than or equal to 3999"`.
```

Comment:
Not sure if this is correct. 

We are saying: if the user puts in a number **greater** than 4000, "Please enter a number less than or equal to 3999." 

it should be saying "Please enter a number less than or equal to 4000" as that is what the task says.

Reply:
It was designed to differentiate itself from https://github.com/freeCodeCamp/freeCodeCamp/blob/ba34c50d499f44e8cedcea30956162b9fda24dca/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-roman-numeral-converter-project/build-a-roman-numeral-converter.md#L94 where it handles the task I'm pretty sure you are referring to. The point was to make sure it handles any random big number greater than 4000. 

Your suggestion, as I understand it, would make require a rewrite of our solution and a few other assertions on this challenge. I am not opposed to updating the wording of this test to more accurately reflect what the test does. However I really wouldn't want to do something out of the scope of adding onto the existing tests.